dev/src/bulk-writer.ts

```diff
@@ -708,29 +867,44 @@ export class BulkWriter {
    * @private
    */
   private sendReadyBatches(): void {
```

Comment:
This API might also be simpler if we passed a pointer with the queue we want to process.